test/test_plugin_helper.rb

```diff
@@ -14,6 +14,8 @@ class ActionController::TestCase
     def add_smart_proxy
       FactoryGirl.create(:smart_proxy, :url => 'http://localhost:8443', :features => [FactoryGirl.create(:feature, :name => 'Openscap')])
       ::ProxyAPI::Features.any_instance.stubs(:features).returns(%w(puppet openscap))
+      versions = { "version" =>"1.11.0", "modules" => { "openscap" => "0.5.3" } }
```

Comment:
nit: missing space before "1.11"
also I'm wondering if we can somehow stub it so it does not break after version bump